docs/resources/ec_deployment.md

```diff
@@ -109,20 +125,21 @@ The optional `kibana.config` and `kibana.topology.config` blocks support the fol
 
 #### APM
 
-The required `apm` block supports the following:
+The optional `apm` block supports the following:
 
-* `topology` - (Required) Topology element which can be set multiple times to compose complex topologies.
+* `topology` - (Optional) Topology element which can be set multiple times to compose complex topologies.
 * `elasticsearch_cluster_ref_id` - (Optional) This field references the ref_id of the deployment Elasticsearch cluster, it is best left to the default value (Defaults to `main-elasticsearch`).
 * `ref_id` - (Optional) ref_id to set on the APM resource. It is best left to the default value (Defaults to `main-apm`).
 * `config` (Optional) APM settings which will be applied to all topologies unless overridden on the topology element. 
 
 ##### Topology
 
-The required `apm.topology` block supports the following:
+The optional `apm.topology` block supports the following:
 
-* `instance_configuration_id` - (Required) Instance Configuration ID from the deployment template. By default, it will use the deployment template default instance configuration, but it can be changed.
-* `memory_per_node` - (Optional) Amount of memory (RAM) per node in the "<size in GB>g" notation. When omitted, it will default to the amount set in the deployment template.
-* `zone_count` - (Optional) Number of zones that the APM deployment will span. This is used to set HA. When omitted, it will default to the amount set in the deployment template.
+* `instance_configuration_id` - (Optional) Instance Configuration ID from the deployment template. By default, it will use the deployment template default instance configuration, but it can be changed. See top level note on `regions and deployment templates`. There's no need to change this value since Kibana only has one _instance type_.
```

Comment:
```suggestion
* `instance_configuration_id` - (Optional) Instance Configuration ID from the deployment template. By default, it will use the deployment template default instance configuration, but it can be changed. See top level note on regions and deployment templates. There's no need to change this value since Kibana only has one _instance type_.
```